packages/fxa-auth-server/lib/senders/emails/sass-compile-files.ts

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import sass from 'sass';
+import { promisify } from 'util';
+import { writeFile, mkdir, existsSync, rmdir, readdirSync } from 'fs';
+import path from 'path';
+
+// promisify the functions in favor of async/await syntax
+const sassRenderPromise = promisify(sass.render);
+const writeFilePromise = promisify(writeFile);
+const makeDirPromise = promisify(mkdir);
+const removeDirPromise = promisify(rmdir);
```

Comment:
You shouldn't need to promisify any of these I don't think - they all have `*Sync` versions: https://gist.github.com/jodyheavener/bf60cafdf6b3dcdd49984c8a31f1bbab